tests/measure.py

```diff
@@ -227,12 +233,14 @@ def testDetection(self):
         # Subtract background
         #
         bgGridSize = 64  # was 256 ... but that gives only one region and the spline breaks
-        bctrl = afwMath.BackgroundControl(afwMath.Interpolate.NATURAL_SPLINE);
-        bctrl.setNxSample(int(self.mi.getWidth()/bgGridSize) + 1);
-        bctrl.setNySample(int(self.mi.getHeight()/bgGridSize) + 1);
+        bctrl = afwMath.BackgroundControl(afwMath.Interpolate.NATURAL_SPLINE)
+        bctrl.setNxSample(int(self.mi.getWidth()/bgGridSize) + 1)
+        bctrl.setNySample(int(self.mi.getHeight()/bgGridSize) + 1)
         backobj = afwMath.makeBackground(self.mi.getImage(), bctrl)
 
-        img = self.mi.getImage(); img -= backobj.getImageF(); del img
+        img = self.mi.getImage()
+        img -= backobj.getImageF()
+        del img
```

Comment:
Does
 
    self.mi.getImage()[:] -= backbone.getImageF()

not work?  I don't think it'll add any overhead